tests/core/test_expectation_suite.py

```diff
@@ -1215,8 +1213,11 @@ def test_add_custom_expectation_emits_event(self, empty_suite):
         suite = empty_suite
 
         class ExpectColumnValuesToBeBetweenOneAndTen(gxe.ExpectColumnValuesToBeBetween):
-            min_value: int = 1
-            max_value: int = 10
+            _min_value = 1
+            _max_value = 10
```

Comment:
Was this required because you removed the `classmethod` decorator?

Reply:
Yes, the validator wasn't working previously, allowing both values to be defaulted to `None`. Removing `classmethod` and fixing the typos made one of the values a requirement during expectation class instantiation in `@root_validator(pre=True)`